curriculum/challenges/english/blocks/workshop-bookstore-page/68ec98b38d83a3c28dd30efe.md

```diff
@@ -28,6 +28,14 @@ assert.equal(document.querySelector('.card h2')?.innerText.trim(), "Sally's SciF
 ## --seed-contents--
 
```

Comment:
Could you format the seed code? For example, for this step, it should be:
```html
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>XYZ Bookstore Page</title>
</head>

<body>
  <h1>XYZ Bookstore</h1>
  <p>Browse our collection of amazing books!</p>
  <div class="card-container">
--fcc-editable-region--
    <div class="card" id="sally-adventure-book">

    </div>
--fcc-editable-region--
  </div>
</body>

</html>
```

This should be done for all the steps.